src/tools/clippy/clippy_lints/src/inherent_impl.rs

```diff
@@ -66,7 +66,7 @@ impl<'tcx> LateLintPass<'tcx> for MultipleInherentImpl {
     }
 
     fn check_crate_post(&mut self, cx: &LateContext<'tcx>, krate: &'tcx Crate<'_>) {
-        if !krate.items.is_empty() {
+        if !krate.items().next().is_some() {
```

Comment:
```suggestion
        if krate.items().next().is_some() {
```

Before, it checked if `items` was **not** empty. This now does the opposite and will enter the if block, if `items()` returns `iter::empty()`.

Reply:
Thanks. I have been looking for this one for too long.